prime-router/metadata/fhir_mapping/hl7/datatypes/XON/Organization.yml

```diff
@@ -127,4 +136,14 @@ extension_3-xon10OrganizationIdentifier:
     valueString:
       expressionType: HL7Spec
       type: STRING
-      valueOf: XON.10
\ No newline at end of file
+      valueOf: XON.10
+
+contact:
+  condition: $XADAddress NOT_NULL
```

Comment:
Let me confirm, but I think this is needed at the top level to prevent
```
contact: [
  {}
]
```

I am thinking it might be worth investigating if this can be changed in the library to prevent empty objects from being added.

Reply:
Similar to the other thread, we should be good without top level conditions as long as all `extension`s in the list have conditions. In this case, we'd still want this condition on `address`, but it frees up the top level to not have to eventually account for name and telecom in a compound condition.

Reply:
Cool, I'll drop them